webui/react/src/components/kit/Modal.tsx

```diff
@@ -52,6 +52,7 @@ interface ModalProps {
   headerLink?: LinkParams;
   icon?: string;
   key?: string;
+  noFooter?: boolean;
```

Comment:
I should probably move this out into the other PR of copy-to-clipboard (it's not going to be worked on until later), but yeah it will be used there (consulted with design). The two were originally done on this branch.